bin/rename-project

```diff
@@ -51,6 +51,38 @@ find . -type f -exec \
     -pe "s/${FIND_MODULE_NAME}/${MODULE_NAME}/g;" {} +
 # -----------------------------------------------------------------------------
 
+cat << 'EOF'
+
+The contents of the /tmp directory need to be deleted because they contain
+references to the old project name and can cause errors if they're not removed.
+
+Some of the files to be deleted are owned by users that only exist in the
+Docker containers, so we need to use `sudo` to delete them. You may be prompted
+for your computer's login password to allow this to happen.
+
+The command that will be run is: "sudo rm -rf ./tmp/* && touch ./tmp/.keep"
+
+You can run this command yourself if you don't want to run it now.
+
+EOF
+
+# -----------------------------------------------------------------------------
+# Delete the contents of the /tmp directory, but not the directory itself or
+# the `.keep` file.
+# -----------------------------------------------------------------------------
+function delete_tmp {
+  sudo rm -rf ./tmp/* && touch ./tmp/.keep
```

Comment:
The `touch`ed `tmp/.keep` does have a different modified time, but it doesn't show up as a modified file in `git`.

Do you think the modified time matters, since the modified time of the rest of the files are either the time when the repo was cloned, or the time when the rename script was run?

Reply:
I think if we leave the `tmp/` directory in tact with the original modified time of the file, it's worth doing the same for the keep file. Something like `find tmp/ ! -name ".keep" -delete` should do the trick but I didn't test it. We would want to make sure the solution works with both the macOS and GNU version of `find`.